docs/metrics/iou.md

```diff
@@ -0,0 +1,170 @@
+# Intersection over Union (IoU)
```

Comment:
Should add this to the top of the file to omit from search results until we land the Metrics Glossary:

```
---
search:
  exclude: true
---
```